source/_integrations/zha.markdown

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Zigbee Home Automation"
-description: "Instructions on how to integrate your Zigbee Home Automation (ZHA) devices within Home Assistant."
+description: "Instructions on how to integrate your Zigbee Home Automation (ZHA) devices within Home Assistant using compatible USB sticks or modules hardware."
```

Comment:
```suggestion
description: "Instructions on how to integrate your Zigbee Home Automation (ZHA) devices within Home Assistant using compatible USB sticks or modules hardware."
```

A title should be short and the point... Please revert.

Reply:
Hope that it is OK with; "Instructions for native integration of Zigbee Home Automation (ZHA) devices in Home Assistant.".